sympy/matrices/dense.py

```diff
@@ -1618,7 +1619,13 @@ def randMatrix(r, c=None, min=0, max=99, seed=None, symmetric=False,
     return m
 
 
-def wronskian(functions, var, method='bareis'):
+def wronskian(functions, var, method='bareiss'):
+    if method == "bareis":
+        SymPyDeprecationWarning(
+                feature="Using 'bareis' to compute matrix determinant",
+                useinstead="'bareiss'",
+                issue=12363, deprecated_since_version="1.1").warn()
+        method = "bareiss"
```

Comment:
This isn't needed here.  If `method='bareis'` is passed in to `wronskian` it will be caught in the `W.det(method)` call.